climada/hazard/test/test_base_xarray.py

```diff
@@ -343,8 +343,8 @@ def test_missing_dims(self):
             hazard = Hazard.from_xarray_raster(ds, "", "")
             self._assert_default_types(hazard)
             np.testing.assert_array_equal(
-                hazard.event_name, [np.datetime64(self.time[0])]
-            )
+                hazard.event_name, [self.time[0].strftime("%Y-%m-%d")]
+                )
```

Comment:
black would format like this
```suggestion
            )
```